minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/ClientStateMachine.java

```diff
@@ -25,17 +25,14 @@
 import java.math.BigDecimal;
 import java.net.InetAddress;
 import java.net.UnknownHostException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Just curious: do we have any norms about when we import things individually vs using a * import?

Reply:
No, convention of Malmo is to explicitly include every sub folder, where Intelij likes to automatically compact them after there are more than n (idk what n is) Happy to disable this if we prefer the other convention

Reply:
Personally I think it's fine to use `*`, but I know Joost said he didn't super like it. Probably just a super low pri thing.